reference/functions/irange.markdown

```diff
@@ -29,19 +29,19 @@ irange(ago(0,0,0,1,30,0), "0");
   * [`ago()`][ago]
   * [`now()`][now]
   * [`accumulated()`][accumulated]
-* Attributes of type irange
-    * [`atime` in body `file_select`][files#atime]
-    * [`copy_size` in body `copy_from`][files#copy_size]
-    * [`ctime` in body `file_select`][files#ctime]
-    * [`mtime` in body `file_select`][files#mtime]
-    * [`match_range` in body `process_count`][processes#match_range]
-    * [`pgid` in body `process_count`][processes#pgid]
-    * [`pid` in body `process_count`][processes#pid]
-    * [`ppid` in body `process_count`][processes#ppid]
-    * [`priority` in body `process_count`][processes#priority]
-    * [`rsize` in body `process_count`][processes#rsize]
-    * [`search_size` in body `file_select`][files#search_size]
-    * [`stime_range` in body `process_count`][processes#stime_range]
-    * [`threads` in body `process_count`][processes#threads]
-    * [`ttime_range` in body `process_count`][processes#ttime_range]
-    * [`vsize` in body `process_count`][processes#vsize]
+* Attributes of type ```irange```
```

Comment:
Why the three ticks instead of one?

Reply:
I see, https://docs.cfengine.com/docs/master/markdown-cheatsheet.html. The triple backticks prevent auto-linking which makes some sense since we already "there" at the irange page.